### DISCUSSION TITLE
强烈 建议把这个例子删了或者换一个

### DISCUSSION THREAD

**Discussion by AllForNothing** [vuejs-translations/docs-zh-cn#261] [General]
<img width="758" alt="image" src="https://user-images.githubusercontent.com/30999793/163509215-d16f995f-3978-4417-9731-8553945f6d06.png">
<img width="790" alt="image" src="https://user-images.githubusercontent.com/30999793/163509301-5a182425-25f0-4910-9d88-70c75e239f6d.png">

作为一个帅气的秃顶程序员，看到这个我气了！！！

**Comment by wxsms** [comment]
![image](https://user-images.githubusercontent.com/5960988/163502764-5d5bea90-5ccd-4f9b-ab0e-0791ec2ad4bd.png)

这貌似只是一个随机回答 yes/no 的 API 哈

**Reply by AllForNothing** [reply]
你说的没错。以上纯属娱乐，Vue 的文档写得很棒！！！！
